lib/Client.js

```diff
@@ -1328,6 +1328,30 @@ class Client extends EventEmitter {
         })));
     }
 
+    /**
+    * Edit guild channels' positions. Note that channel position numbers are lowest on top and highest at the bottom.
+    * @arg {String} guildID The ID of the guild
+    * @arg {Array<Object>} channelPositions An array of [channel positions](https://discord.com/developers/docs/resources/guild#modify-guild-channel-positions)
+    * @arg {String} channelPositions[].id The ID of the channel
+    * @arg {Number} channelPositions[].position The new position of the channel
+    * @arg {Boolean} [channelPositions[].lockPermissions] Whether to sync the permissions with the new parent if moving to a new category
+    * @arg {String} [channelPositions[].parentID] The new parent ID (category channel) for the channel that is moved (only one channel can change category per request)
```

Comment:
> (only one channel can change category per request)

I don't see anything about this in Discord's docs; is there a source for this info I'm missing? Am I blind?

Reply:
That was based on experience because I tried to change more than one channel's category and got the error message stating this.